src/scope/AccessToken.js

```diff
@@ -0,0 +1,74 @@
+import Scope from './Scope'
+import Account from '../entity/Account'
+import ADIOrder from '../entity/ADIOrder'
+import Product from '../entity/Product'
+import Thng from '../entity/Thng'
+import Collection from '../entity/Collection'
+import Action from '../entity/Action'
+import ActionType from '../entity/ActionType'
+import Project from '../entity/Project'
+import PurchaseOrder from '../entity/PurchaseOrder'
+import Redirector from '../entity/Redirector'
+import ShipmentNotice from '../entity/ShipmentNotice'
+import Place from '../entity/Place'
+import File from '../entity/File'
+import AccessPolicy from '../entity/AccessPolicy'
+import AccessTokenEntity from '../entity/AccessToken'
+import Me from '../entity/Me'
+import Access from '../entity/Access'
```

Comment:
Same as for `Operator` scope, not required here.

Reply:
fixed, thanks